packages/extension/src/newtab/MostVisitedSites.tsx

```diff
@@ -24,7 +24,7 @@ export default function MostVisitedSites(): ReactElement {
               href={topSite.url}
               rel="noopener noreferrer"
               className={classNames(
-                'focus-outline w-8 h-8 rounded-lg overflow-hidden bg-white',
+                'focus-outline w-8 h-8 rounded-lg overflow-hidden bg-[#0E1217] border-[#2D323B]',
```

Comment:
Asked our designed and he suggests to use the `secondary-divider` color.
You can use it like so:

```suggestion
                'focus-outline w-8 h-8 rounded-lg overflow-hidden border border-theme-divider-secondary',
```